subscription_manager_setup.host4dhcp.snap.txt

```diff
@@ -0,0 +1,89 @@
+
+# Select package manager for the OS (sets the $PKG_MANAGER* variables)
+if [ -z "$PKG_MANAGER" ]; then
```

Comment:
Feels like something is missing after this

Reply:
The `fi` is in [L40](https://github.com/theforeman/foreman/pull/10166/files#diff-a52fe40bde5850e650b8a2961a4713f6748bf2c91237631e12a01bd17c0acef4R40)

Reply:
yes.